examples/paex_read_write_wire.c

```diff
@@ -105,6 +107,30 @@ int main(void)
     if( err != paNoError ) goto error2;
 
     inputParameters.device = Pa_GetDefaultInputDevice(); /* default input device */
+
+#if USE_LOOPBACK_INPUT
+    for (i = Pa_GetDeviceCount() - 1; i >= 0; i--)
+    {
+        const PaDeviceInfo* device;
+        if ((device = Pa_GetDeviceInfo(i)) != NULL)
+        {
+            // Note: [Loopback] name postfix is provided by the WASAPI device only
+            if (strstr(device->name, "[Loopback]") != NULL)
```

Comment:
1) Host specific code should be avoided in generic examples.

2) Why would we want to use [Loopback] in code that copies input to output?
We end up with a closed loop: output->input->output->input...

Reply:
> 1. Host specific code should be avoided in generic examples.

Ok, I will create a separate file for WASAPI case.

> 2. Why would we want to use [Loopback] in code that copies input to output? We end up with a closed loop: output->input->output->input...

Not really, see the log below. There I play to USB DAC `Neutron HiFi DAC V1` and capture its stream via a Loopback device. Output goes to a 2-nd audio device which is `Speakers (Realtek(R) Audio)`. And it works just fine.

```
patest_read_write_wire.c
sizeof(int) = 4
sizeof(long) = 4
Input device # 5.
    Name: Speakers (2- Neutron HiFi DAC V1) [Loopback]
      LL: 0.003 s
      HL: 0.01 s
Output device # 0.
   Name: Speakers (Realtek(R) Audio)
     LL: 0.0025397 s
     HL: 0.0101587 s
Num channels = 2.
Wire on. Will run 10 seconds.
Wire off.
```

Reply:
Oh! OK, so you are using two different outputs. That makes sense.